rules/game_process.tex

```diff
@@ -75,10 +81,20 @@ \subsection{Robot States}
   \label{fig:robot_states}
 \end{figure}
 
-The referee will announce the transition from the \texttt{initial} to \texttt{ready} state by raising both hands over its head. The referee will wear red gloves \footnote{Teams are encouraged to not rely on the red gloves as they could be removed in a following year.}.
-The referee must stand on the opposite side of the technical area behind the touchline and at the height of the halfway line.
+At the scheduled time, the referee will announce that the transition from the \texttt{initial} to \texttt{visual} will be executed 60 seconds later.
+The referee shall call ``Start in 60 seconds''
+The GameController operator shall start the countdown through the GameController.
```

Comment:
I am not sure if the GameController needs a countdown timer?

Reply:
At least I don't understand this enough to implement it and would rather not do it (it seems an unnecessary extra step to me).

Reply:
The idea is to add consistency between referee and avoiding unfairness of having one teams stiffen while the other team still deploying his code. 
Also, one challenge we discussed about removing some state and having robot allowed to move is it's harder to know when exactly the game start. Making the start time rely less on the referee. 

But I will remove the GC part.